pkgs/by-name/ta/tailscale-gitops-pusher/package.nix

```
@@ -1,10 +1,10 @@
 {
   lib,
   tailscale,
-  buildGo123Module,
+  buildGoModule,
 }:
 
-buildGo123Module {
+buildGoModule {
```

Should we stick with
```suggestion
buildGo124Module {
```
if this is known to break?

---

Good idea.